external/curl/Makefile

```diff
@@ -76,6 +77,7 @@ clean:
 	$(call DELFILE, .built)
 	$(call CLEAN)
 	rm -f $(OBJS)
```

Comment:
Regardless of this change, this line is redundant. From above line, ```call CLEAN```, will delete object files.
Let's remove this line.

Reply:
Okey